packages/relay-kit/src/packs/safe-4337/estimators/generic/types.ts

```diff
@@ -0,0 +1,36 @@
+import { UserOperationStringValues } from '@safe-global/relay-kit/packs/safe-4337/types'
+import { RPC_4337_CALLS } from '@safe-global/relay-kit/packs/safe-4337/constants'
+
+export type GetPaymasterStubDataRpcSchema = [
+  {
+    Method: RPC_4337_CALLS.GET_PAYMASTER_STUB_DATA
+    Parameters: [UserOperationStringValues, string, string, Record<string, any>?]
+    ReturnType:
+      | {
+          paymasterAndData: string
+        }
+      | {
+          sponsor?: { name: string; icon?: string }
+          paymaster?: string
+          paymasterData?: string
+          paymasterVerificationGasLimit?: string
+          paymasterPostOpGasLimit?: string
+          isFinal?: boolean
+        }
+  }
+]
+
+export type PaymasterRpcSchema = [
```

Comment:
Do you really need this being part of the standard?

Reply:
If you remove the generic from createBundlerClient it should work anyways

Reply:
I will remove it.